### DISCUSSION TITLE
tableCell Feature

### DISCUSSION THREAD

**Discussion by ramesh-khatri** [getgauge/taiko#1758] [Ideas]
Hello Everyone, I found this tableCell feature really helpful. I would like to know if there is any feature like getting the value of last row using tableCell({row:-1, col:1}) ? Also is there any feature like getting all the data of specific row of the table and storing it on array? Any suggestion or feedback will be highly appreciated. Thank you :)

**Comment by NivedhaSenthil** [comment]
Both features are not supported currently as a separate api. But `evaluate` api can be used to achieve both cases.

**Reply by ramesh-khatri** [reply]
Thanks, I will try evaluate api.

**Reply by ramesh-khatri** [reply]
@NivedhaSenthil , Is there anything like .then() method. Is evaluate API capable of doing all the things like of .then() ?

**Reply by NivedhaSenthil** [reply]
yes evaluate api returns a promise

**Reply by ramesh-khatri** [reply]
Can't we use node.js push() function in taiko test?

**Reply by NivedhaSenthil** [reply]
which nodejs push do you mean ? callback in evaluate api runs in the browser context, so anything that is specific only to nodejs wont be available there
